### DISCUSSION TITLE
does this work on windows 7?

### DISCUSSION THREAD

**Discussion by ghost** [bjornbytes/lovr#661] [Q&A]
does this work on windows 7 sp1 or older?

**Comment by bjornbytes** [comment]
According to https://lovr.org/docs/Compiling#system-requirements it should work on Vista+, but I'm honestly kinda skeptical of that.  LÖVR requires Vulkan 1.1 now, and I think Vulkan only works on Windows 7 and up.  It would be cool if people with access to older systems could report any issues they have.
